profile/libasn1c.json

```
@@ -0,0 +1,51 @@
+{
+  "Name":           "libasn1c",
+  "SourceUrl":      "https://github.com/osmocom/libasn1c/tags",
+  "Git":            "https://git.osmocom/org/libasn1c",
```

A typo in the domain name
```suggestion
  "Git":            "https://git.osmocom.org/libasn1c",
```